### PR TITLE
Built in components updates #1556

### PR DESCRIPTION
**Pull request by hbutau**
## Description

This PR closes https://github.com/ember-learn/guides-source/issues/1556.

### CODE REVIEW

File: guides/v3.16.0/components/built-in-components.md [hunk 1]
```diff
@@ -1,166 +1,234 @@
-Ember provides a few helpful components out-of-the-box for common tasks,
-including:
+Out of the box, Ember provides 2 components for building a form:
 
-* [`<Input />`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.components/methods/Input?anchor=Input)
-* [`<Textarea />`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
+* [`<Input>`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.components/methods/Input?anchor=Input)
+* [`<Textarea>`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
```
Comment (ijlee2) [inline guides/v3.16.0/components/built-in-components.md:4]: My bad, I forgot that the link doesn't have `v` in front of the number.

```suggestion
* [`<Input>`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.components/methods/Input?anchor=Input)
* [`<Textarea>`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
```

File: guides/v3.16.0/components/built-in-components.md [hunk 2]
```diff
@@ -1,166 +1,234 @@
...
-* `indeterminate`
-* `name`
-* `autofocus`
-* `form`
 
+### Checkboxes
 
-Which can be bound or set as described in the previous section.
+You can use the
+[`<Input>`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.components/methods/Input?anchor=Input)
```
Comment (ijlee2) [inline guides/v3.16.0/components/built-in-components.md:164]:
```suggestion
[`<Input>`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.components/methods/Input?anchor=Input)
```

File: guides/v3.16.0/components/built-in-components.md [hunk 3]
```diff
@@ -1,166 +1,234 @@
...
+  Please present a solution that does not use `{{mut}}`.
+-->
+## Binding dynamic attribute
 
 You might need to bind a property dynamically to an input if you're building a
 flexible form, for example. To achieve this you need to use the
-[`{{get}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/get?anchor=get)
-and [`{{mut}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/mut?anchor=mut)
+[`{{get}}`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.helpers/methods/get?anchor=get)
+and [`{{mut}}`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.helpers/methods/mut?anchor=mut)
```
Comment (ijlee2) [inline guides/v3.16.0/components/built-in-components.md:219]:
```suggestion
[`{{get}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/get?anchor=get)
and [`{{mut}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/mut?anchor=mut)
```

File: guides/v3.16.0/components/built-in-components.md [hunk 4]
```diff
@@ -1,166 +1,234 @@
...
+/>
 ```
 
 The `{{get}}` helper allows you to dynamically specify which property to bind,
 while the `{{mut}}` helper allows the binding to be updated from the input. See
 the respective helper documentation for more detail:
-[`{{get}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/get?anchor=get)
-and [`{{mut}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
+[`{{get}}`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.helpers/methods/get?anchor=get)
+and [`{{mut}}`](https://api.emberjs.com/ember/v3.16/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
```
Comment (ijlee2) [inline guides/v3.16.0/components/built-in-components.md:234]:
```suggestion
[`{{get}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/get?anchor=get)
and [`{{mut}}`](https://api.emberjs.com/ember/3.16/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
```

File: guides/v3.20.0/components/built-in-components.md [hunk 5]
```diff
@@ -1,204 +1,234 @@
-Ember provides a few helpful components out-of-the-box for common tasks,
-including:
+Out of the box, Ember provides 2 components for building a form:
 
-* [`<Input />`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.components/methods/Input?anchor=Input)
-* [`<Textarea />`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
+* [`<Input>`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.components/methods/Input?anchor=Input)
+* [`<Textarea>`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
```
Comment (ijlee2) [inline guides/v3.20.0/components/built-in-components.md:4]:
```suggestion
* [`<Input>`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.components/methods/Input?anchor=Input)
* [`<Textarea>`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
```

File: guides/v3.20.0/components/built-in-components.md [hunk 6]
```diff
@@ -1,204 +1,234 @@
...
-* `indeterminate`
-* `name`
-* `autofocus`
-* `form`
 
+### Checkboxes
 
-Which can be bound or set as described in the previous section.
+You can use the
+[`<Input>`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.components/methods/Input?anchor=Input)
```
Comment (ijlee2) [inline guides/v3.20.0/components/built-in-components.md:164]:
```suggestion
[`<Input>`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.components/methods/Input?anchor=Input)
```

File: guides/v3.20.0/components/built-in-components.md [hunk 7]
```diff
@@ -1,204 +1,234 @@
...
+  Please present a solution that does not use `{{mut}}`.
+-->
+## Binding dynamic attribute
 
 You might need to bind a property dynamically to an input if you're building a
 flexible form, for example. To achieve this you need to use the
-[`{{get}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/get?anchor=get)
-and [`{{mut}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/mut?anchor=mut)
+[`{{get}}`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.helpers/methods/get?anchor=get)
+and [`{{mut}}`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.helpers/methods/mut?anchor=mut)
```
Comment (ijlee2) [inline guides/v3.20.0/components/built-in-components.md:219]:
```suggestion
[`{{get}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/get?anchor=get)
and [`{{mut}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/mut?anchor=mut)
```

File: guides/v3.20.0/components/built-in-components.md [hunk 8]
```diff
@@ -1,204 +1,234 @@
...
+/>
 ```
 
 The `{{get}}` helper allows you to dynamically specify which property to bind,
 while the `{{mut}}` helper allows the binding to be updated from the input. See
 the respective helper documentation for more detail:
-[`{{get}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/get?anchor=get)
-and [`{{mut}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
+[`{{get}}`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.helpers/methods/get?anchor=get)
+and [`{{mut}}`](https://api.emberjs.com/ember/v3.20/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
```
Comment (ijlee2) [inline guides/v3.20.0/components/built-in-components.md:234]:
```suggestion
[`{{get}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/get?anchor=get)
and [`{{mut}}`](https://api.emberjs.com/ember/3.20/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
```

**Comment by ijlee2** [comment]
@hbutau Can you do a find-replace all to remove the letter `v` before the version number in the links to the API Guides?

I documented the issue incorrectly, sorry about that.

**Review by ijlee2 — APPROVED** [review]
@hbutau I went ahead with correcting the URLs, as they were my mistake. Hope that's all right.

I appreciate your working on this issue and contributing to the Ember Guides for your first time!

**Comment by hbutau** [comment]
> @hbutau I went ahead with correcting the URLs, as they were my mistake. Hope that's all right.
> 
> I appreciate your working on this issue and contributing to the Ember Guides for your first time!

Oh okay thank you so much. I am also taking #1560

**Pull request merged**